src/Tasks/GenerateResource.cs

```diff
@@ -2895,14 +2895,14 @@ private string EnsurePathIsShortEnough(string currentOutputFile, string currentO
         /// <param name="filename">Full path to file to delete</param>
         private void RemoveCorruptedFile(string filename)
         {
-            _logger.LogWarningWithCodeFromResources("GenerateResource.CorruptOutput", FileUtilities.GetFullPathNoThrow(filename));
+            _logger.LogMessageFromResources("GenerateResource.CorruptOutput. Attempting to remove possibly-corrupt output file: ", FileUtilities.GetFullPathNoThrow(filename));
             try
             {
                 File.Delete(filename);
             }
             catch (Exception deleteException)
             {
-                _logger.LogWarningWithCodeFromResources("GenerateResource.DeleteCorruptOutputFailed", FileUtilities.GetFullPathNoThrow(filename), deleteException.Message);
+                _logger.LogInfoWithCodeFromResources("GenerateResource.DeleteCorruptOutputFailed. Failed to remove possibly-corrupt output file: ", FileUtilities.GetFullPathNoThrow(filename), deleteException.Message);
```

Comment:
I think you can revert this entirely; if we _fail_ to delete the file we should still log a warning (that should be rare).

Reply:
okay, *should* be fixed